com.unity.netcode.gameobjects/Tests/Runtime/IntegrationTestExamples.cs

```diff
@@ -0,0 +1,192 @@
+using System.Collections;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.TestTools;
+using NUnit.Framework;
+using Unity.Netcode.TestHelpers.Runtime;
+
+namespace Unity.Netcode.RuntimeTests
+{
+    public class IntegrationTestUpdated : NetcodeIntegrationTest
+    {
+        private GameObject m_MyNetworkPrefab;
+        protected override int NbClients => 1;
+
+        protected override void OnServerAndClientsCreated()
+        {
+            m_MyNetworkPrefab = CreateNetworkObjectPrefab("Object");
+            m_MyNetworkPrefab.AddComponent<NetworkVisibilityComponent>();
+        }
+
+        protected override IEnumerator OnServerAndClientsConnected()
+        {
+            SpawnObject(m_MyNetworkPrefab, m_ServerNetworkManager);
+            yield return base.OnServerAndClientsConnected();
+        }
+
+        [UnityTest]
+        public IEnumerator MyFirstIntegationTest()
+        {
+            // Check the condition for this test and automatically handle varying processing
+            // environments and conditions
+            yield return WaitForConditionOrTimeOut(() =>
+            Object.FindObjectsOfType<NetworkVisibilityComponent>().Where(
+                (c) => c.IsSpawned).Count() == 2);
+
+            Assert.False(s_GloabalTimeOutHelper.TimedOut, "Timed out waiting for instances " +
+                "to be detected!");
```

Comment:
is this file a new addition or a refactor/rename?

```suggestion
            yield return WaitForConditionOrTimeOut(() =>
            Object.FindObjectsOfType<NetworkVisibilityComponent>().Where(c => c.IsSpawned).Count() == 2);

            Assert.False(s_GloabalTimeOutHelper.TimedOut, "Timed out waiting for instances to be detected!");
```

Reply:
This file contains the examples within the document for reference purposes.  I can exclude it but figured it could be useful to have as reference and to see that the examples actually work.